openssl-sys/build/find_normal.rs

```diff
@@ -68,6 +77,8 @@ fn find_openssl_dir(target: &str) -> OsString {
         } else if target.contains("apple-darwin") {
             if let Some(dir) = resolve_with_wellknown_homebrew_location("/usr/local/opt/openssl") {
                 return dir.into();
+            } else if let Some(dir) = resolve_with_wellknown_pkgsrc_location("/opt/pkg") {
```

Comment:
There's no need to pass the path in here since there's only one. You can just put it directly in the function.